Add initial runbooks for all critical CMO alerts (#16)
openshift/runbooks: alerts/cluster-monitoring-operator/KubeAPIDown.md

# KubeAPIDown

## Meaning

> The `KubeAPIDown` alert is triggered when the Kubernetes API server
> has not been reachable by the monitoring system for more than 15
> minutes.

Reviewer: ```suggestion
The `KubeAPIDown` alert is triggered when all Kubernetes API servers
have not been reachable by the monitoring system for more than 15
minutes.
```

## Impact

This is a critical alert. The Kubernetes API is not responding. The
cluster may partially or fully non-functional.

## Diagnosis

First, confirm whether the API is also unresponsive for you:

```console
$ oc cluster-info
```

> If you can still reach the API server, there may be an problem with
> only a subset of the API server pods, or there could be a network
> issue. Check the status of the API server pods:

Reviewer: ```suggestion
If you can still reach the API server, there may be a network
issue between the Prometheus instances and the API server pods. Check the status of the API server pods:
```

```console
$ oc -n openshift-kube-apiserver get pods
$ oc -n openshift-kube-apiserver logs -l 'app=openshift-kube-apiserver'
```

> 

Reviewer: ```suggestion
If you can reach the API, check the status of the API server targets in the Prometheus UI.
```
